File: src/main/java/net/lifecity/mc/skillmaster/user/skillset/SkillButton.java
package net.lifecity.mc.skillmaster.user.skillset;

import lombok.AllArgsConstructor;
import lombok.Getter;

/**
 * スキルに使うボタンの列挙
 */
@AllArgsConstructor
public enum SkillButton {
    RIGHT("右クリック"),
    SWAP("スワップ"),
    DROP("ドロップ");

    @Getter
    private final String jp;
}

File: src/main/java/net/lifecity/mc/skillmaster/user/skillset/SkillButton.kt
package net.lifecity.mc.skillmaster.user.skillset

import org.bukkit.Material

enum class SkillButton(val systemName: String, val jp: String, val material: Material) {
    RIGHT("r", "右クリック", Material.YELLOW_DYE),
    SWAP("s", "スワップ", Material.LIGHT_BLUE_DYE),
    DROP("d", "ドロップ", Material.PINK_DYE)
}